prowler/providers/aws/services/awslambda/awslambda_service.py

```diff
@@ -161,7 +161,7 @@ def __list_tags_for_resource__(self):
                 try:
                     regional_client = self.regional_clients[function.region]
                     response = regional_client.list_tags(Resource=function.arn)["Tags"]
-                    function.tags = [response]
```

Comment:
Why are you changing this? This is breaking the standard format of the AWS tagging in Prowler (it has to be a list)

Reply:
Yes, it is how AWS works, in most of the services it return a list of dictionaries where each dictionary has an unique tag. That is why we are forcing it to be a list in this service.

Reply:
That's why we use the unroll_tags function in Prowler for the outputs:
```
        >>> tags = [{"key": "name", "value": "John"}, {"key": "age", "value": "30"}]
        >>> unroll_tags(tags)
        {'name': 'John', 'age': '30'}
```